INSTALL.rst

```diff
@@ -175,4 +239,33 @@ entry points and then do the following (e.g. for the `hep` rule):
 
 .. code-block:: console
 
-    dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep
+    (inspirehep)$ dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep
+
+
+Giving your user admin access
+==============================
+
+If you want access to the admin console, you'll have to execute:
```

Comment:
Did you create a user after installation?

Note that there is a [fixture](https://github.com/inspirehep/inspire-next/blob/cd50b1cbcd028ba6a00d87d676ed0ed187a77382/inspirehep/modules/fixtures/users.py#L35) that creates an initial user with `superuser` role that should be able to access everything.

So either we mention that the user exists, or we teach how to create a new user (and then add the permissions you mention here)?